src/libraries/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/AtomPub10ServiceDocumentFormatterTests.cs

```diff
@@ -516,7 +515,7 @@ public void ReadFrom_FullDocument_ReturnsExpected()
                 Assert.Null(fifthDocument.Language);
                 Assert.Equal(new Uri("http://emptyelement_link.com"), fifthDocument.Link);
 
-                ReferencedCategoriesDocument sixthDocument = Assert.IsType<ReferencedCategoriesDocument>(secondCollection.Categories[5]);
+                Assert.IsType<ReferencedCategoriesDocument>(secondCollection.Categories[5]);
```

Comment:
Same as above, though at least they copied a "document" this time.

Reply:
It also looks like this mistaken code was copied down to line 690 as well, so the same type of change would be needed there.

Reply:
Fixed! Let me know if any of the tests look incorrect. Thanks!